src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -67,13 +54,15 @@ public CommentAttributes(Comment comment) {
         this.showGiverNameTo = comment.getShowGiverNameTo();
         this.showRecipientNameTo = comment.getShowRecipientNameTo();
         this.recipients = comment.getRecipients();
-        this.createdAt = comment.getCreatedAt();
-        this.commentText = comment.getCommentText();
-        this.lastEditorEmail = comment.getLastEditorEmail() != null ?
-                                        comment.getLastEditorEmail() : comment.getGiverEmail();
+        this.lastEditorEmail = comment.getLastEditorEmail() != null ? comment.getLastEditorEmail()
+                                                                    : comment.getGiverEmail();
         this.lastEditedAt = comment.getLastEditedAt() != null ? comment.getLastEditedAt() : comment.getCreatedAt();
     }
 
+    public Long getId() {
+        return getCommentId();
+    }
+    
     public Long getCommentId() {
```

Comment:
There's a bunch of instances like this (you'll see in Db). The primary reason is because the method is defined as abstract in the superclass and had to be implemented, and renaming the existing ones turned out to be problematic because some of them are used in jsp files.
Of course, it is solvable, but not the topmost priority for now.